### ISSUE TITLE
E3111: 'Android' does not contain type or namespace 'android'

### ISSUE THREAD

**Issue by baran** [bolav/fuse-modalview#4]
In fuse 0.31.0 (build 8955) when I try to use ModalView, I got this error message;

```
Fuse 0.31.0 (build 8955)
Build started: FullCompile
Configuring
fuse_modules/bolav/fuse-modalview/AndroidListener.uno(2.15): E3111: 'Android' does not contain type or namespace 'android'. Could you be missing a package reference?
/Users/macmini/workspace/fuse/myProject/fuse_modules/bolav/fuse-modalview/AndroidListener.uno(2,16,2,23): Error E3111: 'Android' does not contain type or namespace 'android'. Could you be missing a package reference?
fuse_modules/bolav/fuse-modalview/ModalJS.uno(8.15): E3111: 'Android' does not contain type or namespace 'android'. Could you be missing a package reference?
/Users/macmini/workspace/fuse/myProject/fuse_modules/bolav/fuse-modalview/ModalJS.uno(8,16,8,23): Error E3111: 'Android' does not contain type or namespace 'android'. Could you be missing a package reference?

Build completed in 7.20 seconds
    2 errors
Build ended
fuse: Failed to compile project
```

**Comment by baran** [comment]
When build for android;

````
Clean completed

Uno 0.42.3 (build 4084) OS X 10.12 i386 b51f27c

Configuring (9.3 s)
Compiling syntax tree
fuse_modules/bolav/fuse-modalview/AndroidListener.uno(23.56): E3111: 'Android' does not contain type or namespace 'java'. Could you be missing a package reference?
fuse_modules/bolav/fuse-modalview/AndroidListener.uno(23.82): E3111: 'Android' does not contain type or namespace 'android'. Could you be missing a package reference?
fuse_modules/bolav/fuse-modalview/AndroidListener.uno(31.25): E3114: There is nothing named 'DialogInterface' accessible in this scope. Are you missing a package reference?
(0.9 s)

Build completed in 10.25 seconds
    3 errors
```

**Comment by bolav** [comment]
This is fixed in 2bfa58d422164c3d32cbc471302dcb687a389b28
